docs/DesignDocs/AttestationQPLEnforcementSupport.md

```diff
@@ -0,0 +1,86 @@
+Attestation QPL Enforcement Support
+====
+
+This proposal is to make a change to `oe_verify_evidence`, so that it can support policy based QPL (Quote Provider Library) enhancement, such as allowing the API user to pass data to QPL for more advanced quote verification.
+
+Motivation
+----
+
+Intel SGX rolls out PCK certificate and collateral information about every six month. There is a scenario that a quote is signed by the last version of PCK certificate cannot be verified with the latest version of collateral information. This causes unnecessary attestation failures when new versions of PCK certificate and collateral information are rolled out. To deal with this issue, the QPL need to be able to return any version collateral information based on additional query parameters (referred as baselines). With that, a new API called `sgx_ql_get_quote_verification_collateral_with_params` is introduced in QPL. Changes are also needed at OE SDK to call this new API.
+
+Goals:
+ - Verifier can call `oe_verify_evidence` and pass QPL specific parameters to do more advanced quote verification.
+
+User Experience
+----
+
+A new policy type `OE_POLICY_COLLATERAL_BASELINE` is added to current policy types. Quote verifier can pass multiple policies with this type, while OE SDK would pass all the policies down to the QPL API without interpreting the data content.
```

Comment:
"Collateral" is not a standard term, not used at https://github.com/openenclave/openenclave/blob/master/docs/DesignDocs/AttestationApiRequirements.md or in the RATS architecture.  It seems to be an Intel SGX specific term, and SGX specific terms shouldn't be used in non-SGX-specific policy types.  If this is SGX specific, then SGX should be in the name like OE_SGX_POLICY_COLLATERAL_BASELINE.  If it's not sgx specific, then it should use a term from the RATS architecture.

Reply:
Good suggestion! let me update the name to be SGX specific,